Troubleshoot/TroubleshootError.ps1

```diff
@@ -231,7 +262,104 @@ else {
     }
 }
 
-if ($LogAnalyticsWorkspaceResourceID -eq $null) {
+
+Write-Host("Currently checking if the cluster is onboarded to custom metrics for Azure monitor for containers...");
+
+#Pre requisite - need cluster spn object Id
+$clusterDetails = Get-AzureRmAks -Id $AKSClusterResourceId -ErrorVariable clusterFetchError -ErrorAction SilentlyContinue;
+if ($clusterFetchError) {
```

Comment:
This call can fail if they don't have Read access to the cluster.

Reply:
updated in PR into ci_feature : abandoning this PR:
https://github.com/Microsoft/OMS-docker/pull/243